cmd/dnsStatusCheck/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.13 AS builder
+ADD . /build
+WORKDIR /build/cmd/dnsStatusCheck
+RUN go build -v
+
+FROM golang:1.13
```

Comment:
lets move the final container to alpine... you may have to build with `GOOS=linux` before this works.  You should run `apk upgrade` to get the newest patches on each build as well.